src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -705,7 +716,10 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
 
             //PROTOTYPE: should be a wellknown member?
             var ctor = type.UnderlyingTupleType?.InstanceConstructors.FirstOrDefault();
-
+            if ((object)ctor == null)
+            {
+                return BadExpression(node);
```

Comment:
@AlekseyTs `BoundTupleCreationExpression` currently requires a non-null constructor (it will assert on null). Is that ok to change to do what you described?